### ISSUE TITLE
Review Accessibility (end of 2022)

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#599]
We should regularly review whether we are getting closer to our accessibility goals. It's of course possible that some changes unknowingly made accessibility in an area worse. As part of this issue, the whole current Tobira should be checked again for various things like:

- Is it possible to navigate by keyboard only? (via tab, enter, ...)
- Does it work well with accessibility tools.
- Does the Wave browser extension report any problems?
- ... (add more here)

Found issues should be documented here and fixed.

**Comment by LukasKalbertodt** [comment]
This issue is outdated. This was done as far as I remember. At the start of '24, a proper accessibility review is planned. Closing as this issue serves no purpose anymore.
